doc/index.md

```diff
@@ -1,14 +1,30 @@
 # Introduction
 
-lua-http is an performant, capable Hyper Text Transfer Protocol (HTTP) and WebSocket (WS) library for Lua 5.1, 5.2, 5.3 and LuaJIT. The software supports x86, x64 and Arm based systems, as well as GNU/Linux, OSX, FreeBSD and others [1]. lua-http can be utilized as a server or client and includes the following features: 
+lua-http is an performant, capable HTTP and WebSocket library for Lua. The library supports Lua 5.1 and greater and runs on a wide range of operating systems and platforms. Some of the features of the library include: 
```

Comment:
I prefer listing the versions of lua: most people just scan the text looking for the thing they care about